test/cloud.jl

```diff
@@ -0,0 +1,13 @@
+using Base.Test
+using Gurobi
+using MathProgBase.SolverInterface
+
+env = Gurobi.Env("csdemo.gurobi.com")
+Gurobi.free_env(env)
```

Comment:
I'm not sure, but the [docs for the client test server](https://www.gurobi.com/documentation/6.5/cloud-guide/client-testing.html) in fact do run a small problem, so I presume it's permitted. Should we copy one from an existing test file, or encapsulate it or something and use it in both the existing test and in this cloud test, or just write a different one here?